src/Preferences/AdvancedPreferencesViewController.m

```diff
@@ -65,7 +65,8 @@ - (NSString *)toolbarItemLabel
  */
 -(IBAction)showAdvancedHelp:(id)sender
 {
-    GotoHelpPage((CFStringRef)@"advanced.html", NULL);
+    NSString *helpBook = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleHelpBookName"];
+    [[NSHelpManager sharedHelpManager] openHelpAnchor:@"AdvancedSettingsSection" inBook:helpBook];
```

Comment:
@Eitot is this working for you? It opens up the help book with this message for me:
![screen shot 2017-02-01 at 17 31 03](https://cloud.githubusercontent.com/assets/799345/22497237/426be416-e8a4-11e6-9d20-cd739c1c11e3.png)